src/EnergyPlus/DataLoopNode.hh

```diff
@@ -262,12 +309,26 @@ namespace DataLoopNode {
 			TempSetPointHi( TempSetPointHi ),
 			TempSetPointLo( TempSetPointLo ),
 			Height( Height ),
+			SchedOutAirDryBulb ( SchedOutAirDryBulb ),
```

Comment:
I only saw this one with the original constructor and is actually used in the DataLoopNode.cc
They have a note like:
```
NodeData DefaultNodeValues( 0, 0, 0.0, 0.0, 0.0, SensedNodeFlagValue, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, SensedNodeFlagValue, SensedNodeFlagValue, SensedNodeFlagValue, SensedNodeFlagValue, SensedNodeFlagValue, -1.0, 0.0, false, 0.0, 0.0, false, 0.0, 0.0, 0.0, 0.0, 0.0, false ); 
//Autodesk:Note If intent is default construction drop initializer to elim bug exposure | FluidType | FluidIndex | Temp {C} | TempMin {C} | TempMax {C} | TempSetPoint {C} | TempLastTimeStep {C} | MassFlowRateRequest {kg/s} | MassFlowRate {kg/s} | MassFlowRateMin {kg/s} | MassFlowRateMax {kg/s} //Autodesk:Note SensedNodeFlagValue is default initializer | MassFlowRateMinAvail {kg/s} | MassFlowRateMaxAvail {kg/s} | MassFlowRateSetPoint {kg/s} | Quality {0.0-1.0 vapor fraction/percent} | Press {Pa}   REAL(r64)     :: | Enthalpy {J/kg} | EnthalpyLastTimeStep {J/kg} | HumRat {} | HumRatMin {} | HumRatMax {} | HumRatSetPoint {} | TempSetPointHi {C} | TempSetPointLo {C} | Height {m} | OutAirDryBulb {C} | EMSOverrideOutAirDryBulb | EMSValueForOutAirDryBulb {C} | OutAirWetBulb {C} | EMSOverrideOutAirWetBulb | EMSValueForOutAirWetBulb {C} | CO2 {ppm} | CO2 setpoint {ppm} | Generic contaminant {ppm} | Generic contaminant setpoint {ppm} | Set to true when node has SPM which follows wetbulb	
```